.github/workflows/format.yml

```diff
@@ -0,0 +1,15 @@
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/prepare
+      - run: npm run format:check
+
+name: Format
+
+on:
+  pull_request: ~
```

Comment:
TIL that `~` means `null` in YAML. Perhaps just write it as `null` for clarity?

Is this saying that we don't want to run format checks on PR branches? Why would that be?

Reply:
Heh I went the opposite conclusion in https://github.com/JoshuaKGoldberg/template-typescript-node-package - that it implies _"run on all of them"_, which `null` feels ...  the opposite to me?

It means we run format checks on:
* All pull request commits
* Commits to `main`